config/prow/plugins.yaml

```diff
@@ -991,6 +991,8 @@ cherry_pick_unapproved:
 
     To merge this cherry pick, it must first be approved (`/lgtm` + `/approve`) by the relevant OWNERS.
 
+    If you didn't cherry-pick this change to [**all supported release branches**](https://k8s.io/releases/patch-releases), please leave a comment describing why other cherry-picks are not needed to speed up the review process.
```

Comment:
do we want to add this guidance to https://github.com/kubernetes/community/blob/master/contributors/devel/sig-release/cherry-picks.md as well, maybe with some examples of why someone might not apply the fix to all releases (bug didn't exist until 1.x, bug was in alpha-gated code prior to 1.x, fix didn't apply cleanly or was higher risk prior to 1.x, etc)

Reply:
That's a very good point. Updated the PR.